apps/rod/views/reuse-main-applicant-address.html

```diff
@@ -0,0 +1,21 @@
+{{<partials-page}}
+  {{$page-content}}
+  {{#markdown}}delivery-address-for-documents{{/markdown}} 
+
+  <div class="govuk-details__text">
+  <div class="govuk-summary-list__row summary-list-box" >
```

Comment:
This class may cause an issue in the smaller screens with an underline. I don't think we need this wrapper here.